packages/flutter/lib/src/material/typography.dart

```diff
@@ -493,6 +500,48 @@ class Typography with Diagnosticable {
     overline  : TextStyle(debugLabel: 'whiteCupertino overline',  fontFamily: '.SF UI Text',    inherit: true, color: Colors.white,   decoration: TextDecoration.none),
   );
 
+  /// A material design text theme with dark glyphs based on San Francisco.
+  ///
+  /// This [TextTheme] provides color but not geometry (font size, weight, etc).
+  ///
+  /// This theme uses the macOS version of the font names.
+  static const TextTheme blackRedwoodCity = TextTheme(
+    headline1 : TextStyle(debugLabel: 'blackCupertino headline1', fontFamily: '.AppleSystemUIFont', inherit: true, color: Colors.black54, decoration: TextDecoration.none),
```

Comment:
Shouldn't the debug label be "blackRedwoodCity headline1" and below too?

Reply:
Yup